### PR TITLE
[Ready] [Style] 0.43 Style issues bundle 

### PR DESCRIPTION
**Pull request by caneruguz**
## Purpose
Fixes the following Trello cards:

https://trello.com/c/4RpR6p0R
https://trello.com/c/dXX6OioD
https://trello.com/c/cSzGd1ya
https://trello.com/c/CWj6sY7R
https://trello.com/c/zNY34VaB
https://trello.com/c/t9y7mTVi
https://trello.com/c/imvv0JJs

### CODE REVIEW

File: website/templates/util/render_node.mako [hunk 1]
```diff
@@ -119,8 +119,8 @@
 % else:
     <li
         node_reference="${summary['id']}:${'node' if summary['primary'] else 'pointer'}"
-        class="project list-group-item list-group-item-node unavailable">
```
Comment (haoyuchen1992) [inline website/templates/util/render_node.mako]: Hi @caneruguz , I added a style that depends on `unavailable`  class -- https://github.com/caneruguz/osf.io/blob/stylefix/deferred-cards-3/website/static/css/style.css#L430 . I used the grey background to show that private component is not clickable.

Comment (sloria) [inline website/templates/util/render_node.mako]: @caneruguz Was that class purposefully removed?

Comment (caneruguz) [inline website/templates/util/render_node.mako]: @haoyuchen1992 That style is the cause of the Trello card. Having background be dark makes it look like a header or footer in that list item group. 

@sloria yes that is intentional. I should have removed the style as well that Haoyu is referring to. I can push that as well if you want. 

As long as unavailable is not being used by js it should be fine, and I don't think it gets a use like that.

Comment (sloria) [inline website/templates/util/render_node.mako]: Ah, just finished reading this before I merged. Yes, if `unavailable` is unused, let's remove it.

**Pull request merged**